checkout-extension/src/Checkout.liquid

```diff
@@ -1,33 +1,38 @@
 {%- if flavor contains "react" -%}
-import React from 'react';
```

Comment:
The React extensions use the `jsx-runtime` React transform, which does not require this value to be in scope. There's actually a warning about this in source code when generating an extension today.